src/main/java/org/codingkata/week3/StringManipulation.java

```diff
@@ -0,0 +1,17 @@
+package org.codingkata.week3;
+
+public class StringManipulation {
+
+    public static int getStringLength(String input){
+        int length =0;
+        for (char c : input.toCharArray()){
+            length++;
+        }
+        return length;
+
```

Comment:
Please remove this empty row and format the code. You can see that at line 6, there is no space between `=` and `0`.